plasma/plasma-mvp/specifications/morevp.md

```diff
@@ -0,0 +1,648 @@
+Multiple competing transactions can be revealed during this period, but only the oldest presented transaction is considered for the purposes of a response.
+
+If any transactions have been presented during the first period, any other user can respond to the challenge by proving that the exiting transaction is actually included in the chain before the oldest presented competing transaction.
+If this response is given, then the exiting transaction is determined to be canonical and the responder receives the `exit bond` placed by the user who started the exit.
+Otherwise, the exiting transaction is determined to be non-canonical and the challenger receives `exit bond`.
+
+Note that this challenge means it’s possible for an honest user to lose `exit bond` as they might not be aware their transaction is non-canonical.
+We address this attack vector and several mitigations in detail later.
+
+
+#### Joining an Exit
+
+As noted earlier, it’s possible that some participants in a transaction may not be aware that the transaction is non-canonical.
+Owners of both inputs and outputs to a transaction may want to start an exit in the case that they would receive the funds from the exit.
+However, we want to avoid the gas cost of repeatedly publishing and proving statements about the same transaction.
+We therefore allow owners of inputs or outputs to a transaction to join (“piggyback”) an existing exit that references the transaction.
+This effectively “caches” the result of the “exitable” and “canonical” checks.
+
+Users must join an exit within the first period.
+To join an exit, an input or output owner places a bond, `piggyback bond`.
+This bond is used to cover the cost of challenges that show the input or output is spent.
+A successful challenge blocks the specified input or output from exiting.
+These challenges must be presented before the end of the second period.
+
+Note that it isn’t mandatory to piggyback an exit.
+Users who choose not to join an exit are choosing not to attempt a withdrawal of their funds.
+If the chain is byzantine, not piggybacking could potentially mean loss of funds.
+
+
+### Attack Vectors and Mitigations
+
+#### Honest Exit Bond Slashing
+
+It’s possible for an honest user to start an exit and have their exit bond slashed.
+This can occur if one of the inputs to a transaction is malicious and signs a second transaction spending the same input.
+
+The following scenario demonstrates this attack: 
+1. Mallory spends `UTXO1` in `TX1` to Bob, creating `UTXO2`.
+2. `TX1` is in-flight.
+3. Operator begins withholding blocks while `TX1` is still in-flight.
+4. Bob starts an exit referencing `TX1` and places `exit bond`.
+5. Mallory spends `UTXO1` in `TX2`.
+6. In period 1 of the exit for `TX1`, Mallory challenges the canonicity of `TX1` by revealing `TX2`.
+7. No one is able to respond to the challenge in period 2, so `TX1` is determined to be non-canonical.
+8. After period 2, Mallory receives Bob’s `exit bond`.
```

Comment:
in the section where you describe the canonicity game, I think it is implied that that happens after period 1 not 2

Reply:
Updated the canonicity game with the line "If this response is given before the end of the second period..." to make clear that Mallory receives the bond after period 2.